main/src/main/java/cgeo/geocaching/unifiedmap/mapsforgevtm/MapsforgeThemeHelper.java

```diff
@@ -192,6 +192,10 @@ protected void disposeTheme() {
         if (mTheme != null) {
             mTheme.dispose();
         }
+        // reset scale to avoid tile scaling issues (see #13593)
```

Comment:
This will disable all scaling effects for VTM-based maps effectively. Shouldn't we hide the related settings in theme options then as well (for the time being), until this is fixed in VTM?

Reply:
> This will disable all scaling effects for VTM-based maps effectively.  

This is just called while disposing the theme. As soon as the theme is loaded, `applyScales(...)` will take care of setting the scales again.

Have you tested if scaling really doesn't work? If yes, there would be a bug inside the implementation as it's at least supposed to work correctly...

Reply:
This is how it looks currently:

|Scaling options|NewMap|VTM|
|---|---|---|
|![image](https://github.com/cgeo/cgeo/assets/3754370/acdfb21e-f71b-4cfd-9609-e92f3345841a)|![image](https://github.com/cgeo/cgeo/assets/3754370/65930353-96b3-4fd0-87cc-4a4cdf94e093)|![image](https://github.com/cgeo/cgeo/assets/3754370/2b4ba186-72d8-474c-ba3b-afb33db0bf66)|

Reply:
This is how it looks on VTM using current master, without this PR:

Not exactly useful, but it DOES do some scaling :-)

![image](https://github.com/cgeo/cgeo/assets/3754370/181dea53-aee7-4667-9a38-e3c9f47f2c1d)

(Differences to NewMap screenshot may be due to different map position and/or different theme settings besides scaling options, have not cross-checked.)

Reply:
Ok, indeed this doesn't look right. If we doesn't find a different solution I would just vote for dropping the scaling options... (still better than wrongly placed tiles)